src/styles/utils/mixin.scss

```diff
@@ -0,0 +1,23 @@
+@mixin number-day {
+  @for $cell from 1 through 31 {
+    & :nth-child(#{$cell})::before {
+      content: '#{$cell}';
+    }
+  }
+}
+
+@mixin weak-day-lentgh {
+  @each $week-day, $index in $start-day {
+    &--start-day-#{$week-day} :nth-child(1) {
+      margin-left: $cellSize * $index + $index;
+    }
+  }
+}
```

Comment:
It seems there is a typo in the mixin name `weak-day-lentgh`, it should be `week-day-length` to correctly represent its purpose.